static/app/views/performance/transactionSummary/transactionAnomalies/anomalyChart.tsx

```diff
@@ -53,8 +53,8 @@ const _AnomalyChart = (props: Props) => {
     xAxis: undefined,
     yAxis: {
       axisLabel: {
-        // p50() coerces the axis to be time based
-        formatter: (value: number) => axisLabelFormatter(value, 'p50()'),
```

Comment:
won't the ADS service response be an overall count here? (e.g. summed for the entire time bucket rather than TPM)

Reply:
we could also just have the ADS service use TPM instead of count if that's better

Reply:
It's fine, we're just using the name to force the labelFormatter to know it's `count` for now, we need to fix this in event-stats in general but that's not going to be in this PR.

Reply:
ooo right that makes sense, cool